src/pages/home/report/EmojiPickerMenu/index.js

```diff
@@ -61,6 +91,138 @@ class EmojiPickerMenu extends Component {
         if (this.props.forwardedRef && _.isFunction(this.props.forwardedRef)) {
             this.props.forwardedRef(this.searchInput);
         }
+
+        // Setup and attach keypress/mouse handlers only if we have a keyboard (and not a touchscreen)
+        // NOTE: these event handlers are instance members so we can reference
+        // the same functions in memory when removing them.
+        if (!canUseTouchScreen() && document) {
+            this.keyDownHandler = (e) => {
+                // Move the highlight when arrow keys are pressed
+                if (e.key.startsWith('Arrow')) {
+                    this.highlightAdjacentEmoji(e.key);
+
+                    // Depending on the position of the highlighted emoji after moving,
+                    // toggle which arrow keys can affect the cursor position in the search input.
+                    this.toggleArrowKeysOnSearchInput();
+                }
+
+                // Select the highlighted emoji if enter is pressed
+                if (e.key === 'Enter' && this.state.highlightedIndex !== -1) {
+                    this.props.onEmojiSelected(this.state.filteredEmojis[this.state.highlightedIndex].code);
+                }
+            };
+            document.addEventListener('keydown', this.keyDownHandler);
+
+            // Re-enable pointer events and hovering over EmojiPickerItems when the mouse moves
+            this.mouseMoveHandler = () => {
+                if (this.state.shouldDisablePointerEvents) {
+                    this.setState({shouldDisablePointerEvents: false});
+                }
+            };
+            document.addEventListener('mousemove', this.mouseMoveHandler);
+        }
+    }
+
+    componentWillUnmount() {
+        // Cleanup all mouse/keydown event listeners that we've set up
+        if (!canUseTouchScreen() && document) {
+            document.removeEventListener('keydown', this.keyDownHandler);
+            document.removeEventListener('keydown', this.mouseMoveHandler);
+            this.toggleKeysOnSearchInput([], ['ArrowDown', 'ArrowLeft', 'ArrowRight', 'ArrowUp']);
+        }
+    }
+
+    /**
+     * Highlights emojis adjacent to the currently highlighted emoji depending on the arrowKey
+     * @param {String} arrowKey
+     */
+    highlightAdjacentEmoji(arrowKey) {
```

Comment:
Looking back the name of this function is a bit confusing; its purpose is just to move the highlight to a new index depending on which arrow key is pressed. Do you guys have any recommendations on better names (if you think a name change is necessary).

@pecanoro @stitesExpensify

Reply:
Nah, I think the name is pretty good, I would leave it like this